elliottlib/cli/advisory_commons_cli.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from elliottlib import logutil
+from elliottlib.cli.common import cli
+from elliottlib.util import green_prefix
+from errata_tool import Erratum, ErrataException
+import click
+
+LOGGER = logutil.getLogger(__name__)
+supported_update_fields = [
+    'publish_date',
+    'text_only',
+    'text_only_cpe',
+    'owner_email',
+    'manager_email',
+    'manager_id',
+    'qe_email',
+    'qe_group',
+]
+
+
+@cli.command("advisory-commons", short_help="Show or update advisory commons for a group of advisories")
+@click.option("--advisories", metavar='ADVISORY',
+              help="Select given advisory id(s). comma separated")
+@click.option("--field", required=True,
+              help="Advisory field (like publish_date) to show or update. Examples of show only fields: ["
+                   "'errata_state', "
+                   "'errata_name', 'content_types', 'synopsis', "
+                   "'errata_bugs', "
+                   "'errata_type']")
+@click.option("--new",
+              help=f"New value of field to update advisories with. --field should be one of"
+                   f" {supported_update_fields}")
```

Comment:
Would like to have separate options for each supported field for better validation, help message, and being able to update multiple fields.

Reply:
Maybe next hack n hustle :sweat_smile: ?
Don't want to spend too much time on this right now.. date format is validated in errata_tool which I think is sufficient for the initial implementation

Reply:
Added a --version-replace option specifically for topic, synopsis and description